platform/linux-generic/_fdserver.c

```diff
@@ -4,6 +4,10 @@
  * SPDX-License-Identifier:     BSD-3-Clause
  */
 
+#ifdef HAVE_CONFIG_H
```

Comment:
HAVE_CONFIG_H is always enabled now. Or I do not see a case where it is disabled.